docs/current/cli/979595-reference.md

```diff
@@ -214,6 +398,79 @@ Disable Dagger terminal output, but continue emitting program standard output:
 dagger --silent run go run ci.go 2>&1 | tee foo.out
 ```
 
+## dagger shell
+
+:::note
+This command is currently under development and is therefore hidden in the CLI.
+:::
+
+Open a shell in a container returned by a function. If no entrypoint is specified and the container doesn't have a default command, `sh` will be used.
+
+### Usage
+
+```shell
+dagger shell [--entrypoint string] [function]
+```
+
+### Options
+
+| Option                 | Description                  |
+| ---------------------- | -----------------------------|
+| `--entrypoint command` | Entrypoint to use            |
+
+### Example
+
+Open a shell session in the container returned by the `debug` function:
+
+```shell
+dagger shell debug
+```
+
+Open a shell session in the container returned by the `debug` function using a different entrypoint:
+
+```shell
+dagger shell --entrypoint /bin/zsh debug
+```
+
+## dagger up
+
+:::note
+This command is currently under development and is therefore hidden in the CLI.
+:::
+
+Start a service or a container returned by a function and expose its ports to the host.
```

Comment:
It only works on services, not containers.

Reply:
Modified, also in quickstart